src/js/elements/mapchip/components/filter_label.js

```diff
@@ -94,9 +94,21 @@ export class FilterLabel extends Component {
         this._filterHeaderLabelContainer.removeClass("notification-badges");
     }
 
-    compareFilters(previouslySelectedFilters, oldFilters) {
-        const showBadge = this.isEqualsJson(previouslySelectedFilters, (oldFilters[0]?.filters || []));
-        if (!showBadge) {
+    compareFilters(previouslySelectedFilters, oldFilters, siteWideFilters) {
+        let newFiltersClone = structuredClone(previouslySelectedFilters);
+        newFiltersClone.forEach(f => {
+            f.isSiteWideFilter = siteWideFilters.filter(x => x.indicatorValue === f.group && x.subIndicatorValue === f.value)[0] != null;
```

Comment:
suggestion: I think it will be better to use `.length > 0` rather than using  `[0] != null`